demisto_sdk/commands/common/tests/integration_test.py

```diff
@@ -2683,3 +2683,83 @@ def test_is_native_image_does_not_exist_in_yml_fail(
             integration_validator.is_native_image_does_not_exist_in_yml()
             == is_validation_ok
         )
+
+    VALID_COMMAND_OUTPUTS = [
+        {
+            "contextPath": "URL.Data",
+            "description": "test description.",
+            "type": "string",
+        },
+        {
+            "contextPath": "Domain.Metadata",
+            "description": "metadata",
+            "type": "string",
+        },
+        {
+            "contextPath": "InfoFile.Metadata",
+            "description": "metadata",
+            "type": "string",
+        },
+        {
+            "contextPath": "IP.Metadata",
+            "description": "metadata",
+            "type": "string",
+        },
+        {
+            "contextPath": "File.Metadata",
+            "description": "metadata",
+            "type": "string",
+        },
+        {
+            "contextPath": "CVE.Metadata",
+            "description": "metadata",
+            "type": "string",
+        },
+        {
+            "contextPath": "Email.Metadata",
+            "description": "metadata",
+            "type": "string",
+        },
+        {
+            "contextPath": "Endpoint.Metadata",
+            "description": "metadata",
+            "type": "string",
+        },
+        {
+            "contextPath": "Certificate.Metadata",
+            "description": "metadata",
+            "type": "string",
+        },
+    ]
+    INVALID_COMMAND_OUTPUTS = [
```

Comment:
what's invalid in each?

Reply:
Url and Cve are invalid with lowercase letters.